tools/biolatency.py

```diff
@@ -15,39 +15,51 @@
 from bcc import BPF
 from time import sleep, strftime
 import argparse
+import json
+import os.path
 
 # arguments
 examples = """examples:
-    ./biolatency            # summarize block I/O latency as a histogram
-    ./biolatency 1 10       # print 1 second summaries, 10 times
-    ./biolatency -mT 1      # 1s summaries, milliseconds, and timestamps
-    ./biolatency -Q         # include OS queued time in I/O time
-    ./biolatency -D         # show each disk device separately
-    ./biolatency -F         # show I/O flags separately
+    ./biolatency                    # summarize block I/O latency as a histogram
+    ./biolatency 1 10               # print 1 second summaries, 10 times
+    ./biolatency -mT 1              # 1s summaries, milliseconds, and timestamps
+    ./biolatency -Q                 # include OS queued time in I/O time
+    ./biolatency -D                 # show each disk device separately
+    ./biolatency -F                 # show I/O flags separately
+    ./biolatency -j                 # print a dictionary
+    ./biolatency -j -f name.json    # Dump the histogram dictionary to name.json file
 """
 parser = argparse.ArgumentParser(
     description="Summarize block device I/O latency as a histogram",
     formatter_class=argparse.RawDescriptionHelpFormatter,
     epilog=examples)
 parser.add_argument("-T", "--timestamp", action="store_true",
-    help="include timestamp on output")
+                    help="include timestamp on output")
```

Comment:
Could you avoid format change here so we can avoid unnecessary code churn? The same for other places in this commit.

Reply:
Yes, I'll fix all of that!